src/js/utils/UserSettings.js

```diff
@@ -39,7 +40,8 @@
         maintainRatio : true,
         resizeContent : false,
         origin : 'TOPLEFT'
-      }
+      },
+      COLOR_FORMAT: 0,
```

Comment:
I used an array for easy converting from the 0 based index that you get from the select input. I was trying to mimic how the GRID_WIDTH setting works, but I couldn't find the place that it's index is changed into something more representative.

I'll dig into this one tomorrow and push once I'm done.